### DISCUSSION TITLE
【委托】求设备

### DISCUSSION THREAD

**Discussion by jsdhd1max** [share121/inter-knot#5005] [委托]
开学了，想在学校追番，有什么手机以外的追番设备吗？希望大佬们推荐一下

**Comment by luo3323** [comment]
小天才

**Reply by Mollsity** [reply]
不是😡什么4k高清

**Comment by Mollsity** [comment]
偷偷带个pad过去😋

**Comment by passbyDFX** [comment]
希沃白板doge

**Comment by manyshoes** [comment]
mp3，十年前就有那种平价支持视频的MP3了，以前是缓存小说和老番剧带到学校看，不过手机都不能用的环境，我的建议是好好学习

**Comment by cyq20091029** [comment]
可以去海鲜市场淘个OPPOwatch之类的，到时候再用adb装播放器<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
